messages/auth.json

```diff
@@ -9,5 +9,7 @@
   "PortInUseAction": "Kill the process running on port %s or use a custom connected app and update OauthLocalPort in the sfdx-project.json file.",
   "invalidRequestMethod": "Invalid request method: %s",
   "invalidRequestUri": "Invalid request uri: %s",
-  "pollingTimeout": "The device authorization request timed out. After executing force:auth:device:login, you must approve access to the device within 10 minutes.  This can happen if the URL wasn’t copied into the browser, login was not attempted, or the 2FA process was not completed within 10 minutes. Request authorization again."
+  "pollingTimeout": "The device authorization request timed out. After executing force:auth:device:login, you must approve access to the device within 10 minutes.  This can happen if the URL wasn’t copied into the browser, login was not attempted, or the 2FA process was not completed within 10 minutes. Request authorization again.",
+  "serverErrorHTMLResponse": "<h1>%s</h1><br />This is most likely <b>not</b> an error with the Salesforce CLI. Please ensure all information is accurate and try again.",
```

Comment:
Edit: "This is most likely <b>not</b> an error with Salesforce CLI. Ensure all information is accurate and try again."